tests/start-cypress-tests.sh

```diff
@@ -30,6 +30,22 @@ fi
 echo "Logging into Kube API server..."
 oc login --server=$CYPRESS_OC_CLUSTER_URL -u $CYPRESS_OC_CLUSTER_USER -p $CYPRESS_OC_CLUSTER_PASS --insecure-skip-tls-verify
 
+echo "Checking RedisGraph deployment."
+oc get pod -n open-cluster-management | grep search-redisgraph-0 | grep Running
+if [ "$?" == "1" ]; then
+  echo "RedisGraph not found, deploying and waiting 60 seconds for the search-redisgraph-0 pod."
```

Comment:
This code will also run if search-redisgraph-0 is found but not running. Might be helpful to log the actual state here.

Reply:
@SherinV Now that we have the `deployredisgraph` status in our search operator, should use that instead of the pod status?